test/mock-server/deps.ts

```diff
@@ -1,18 +1,18 @@
-export type { Header } from 'https://deno.land/x/djwt/mod.ts'
+export type { Header } from 'https://deno.land/x/djwt@v2.4/mod.ts'
```

Comment:
I see that the last commit on this file did the opposite and remove the hardcoded version.

Reply:
It's never a good idea to use latest or a `*` dependency. Will lead to errors like this: https://github.com/onfido/onfido-sdk-ui/runs/5036445484?check_suite_focus=true